.github/workflows/CICD.yml

```diff
@@ -894,6 +898,21 @@ jobs:
         case '${{ matrix.job.os }}' in
           macos-latest) brew install coreutils ;; # needed for testing
         esac
+        case '${{ matrix.job.os }}' in
+          *linux*)
+            # pinky is a tool to show logged-in users from utmp, and gecos fields from /etc/passwd.
+            # In GitHub Action *nix VMs, no accounts log in, even the "runner" account that runs the commands. The account also has empty gecos fields.
+            # To work around this for pinky tests, we create a fake login entry for the GH runner account...
+            FAKE_UTMP='[7] [999999] [tty2] [runner] [tty2] [] [0.0.0.0] [2022-02-22T22:22:22,222222+00:00]'
+            # ... by dumping the login records, adding our fake line, then reverse dumping ...
+            (utmpdump /var/run/utmp ; echo $FAKE_UTMP) | sudo utmpdump -r -o /var/run/utmp
+            # ... and add a full name to each account with a gecos field but no full name.
+            sudo sed -i 's/:,/:runner name,/' /etc/passwd
+            # We also create a couple optional files pinky looks for
+            touch /home/runner/.project
+            echo "foo" > /home/runner/.plan
```

Comment:
I tried to get a little bit better code coverage by adding this, but it doesn't seem to have worked. I don't know why, pretty much the same lines in [my CI sandbox](https://github.com/jtracey/actions-sandbox/runs/5354652360?check_suite_focus=true#step:20:6) seem to work just fine.

Reply:
If nobody knows, I can just back it out for now and keep those lines tested.